packages/-ember-data/tests/integration/adapter/queries-test.js

```diff
@@ -76,7 +76,7 @@ module('integration/adapter/queries - Queries', function(hooks) {
     let queryResults = await store.query('person', { page: 1 });
 
     assert.equal(queryResults.length, 2, 'the record array has a length of 2 after the results are loaded');
-    assert.equal(queryResults.isLoaded, true, "the record array's `isLoaded` property should be true");
+    assert.true(queryResults.isLoaded, "the record array's `isLoaded` property should be true");
```

Comment:
Btw, whats the advantage of assert true vs equal to true?

Reply:
huh, apparently the comment I left didn't save :/ 

basically its just a newer thing that's more explicit. There's a lint rule that auto fixes to this thing. I don't care a ton but I do find the new style a little quicker to read and since there's an autofix I'm all for it.

Reply:
(comment meaning my first reply to you here)